jekyll/_cci2/faq.md

```diff
@@ -248,6 +248,86 @@ Yes, the billing is associated with the organization. You can buy while within t
 {:.no_toc}
 A container is a 2 CPU 4GB RAM machine that you pay for access to. Containers may be used for concurrent tasks (for example, running five different jobs) or for parallelism (for example, splitting one job across five different tasks, all running at the same time). Both examples would use five containers.
 
+### What are credits?
+{:.no_toc}
+
+Credits are used to pay for your usage based on machine type and size.
+
+For example, the 25,000 credit package would provide 2,500 build minutes when
+using a single machine at the default rate of 10 credits per minute. The same package would last 1,250 minutes when using 2x parallelism or 250 minutes at 10x parallelism.
+
+### Can I get one workflow run to use 50 containers at once without queuing?
+{:.no_toc}
+
+The Performance plan will experience less queuing, so you may run 50x parallelism builds and kick off a bunch concurrently, but this will increase your credit usage significantly.  The Seed plan has a maximum of 4x parallelism.
+
+### If a container is used for under one minute, do I have to pay for a full minute? 
+{:.no_toc}
+
+You pay to the nearest credit. First we round up to the nearest second, and then up to the nearest credit. On the 2x4 (medium) box this is effectively in 6 second increments. 
+
+### How do I buy credits? Can I buy in any increments?
+{:.no_toc}
+
+Every month, you are charged for your selected credit package at the beginning of the month.
+Bonus credits are offered on the Performance plan for 50,000 credit purchases and above.
+
+### What happens when I run out of credits?
+{:.no_toc}
+
+On the Seed plan, you never have to worry about compute spend going above your budget. Your projects will not build until your billing cycle is complete. You will get an error message in-app and through email explaining why your build did not run. If you decide you want to build more, you can upgrade to a Performance plan if you choose.
+
+On the Performance plan, whenever you run out of credits, you can elect to have your plan renew so builds will never be blocked on a lack of credits in your account.
+
+### What happens to my credits if I upgrade to a new plan?
+{:.no_toc}
+
+If you upgrade within Seed, the rates are exactly the same.
+If you upgrade to Performance, credits rollover from your Seed plan.
+
+### Do credits expire?
+{:.no_toc}
+
+**Hobbyist**: Credits expire every week.
```

Comment:
Just want to make sure that we're aligned with Rose. We've been calling them "Free", "Basic", and "Performance". I want to make sure we're all on the same page with naming our plan tiers.

Reply:
Will make the change unless noted otherwise.